### ISSUE TITLE
Please expain how to make compatible keys:

### ISSUE THREAD

**Issue by AndKe** [ronf/asyncssh#417]
I tried to use this example:
https://asyncssh.readthedocs.io/en/latest/#id14

Generated keys with password, using ssh-keygen, but than I get:

```
asyncssh.pbe.KeyEncryptionError: OpenSSH private key encryption requires bcrypt with KDF support
```

How can I satisfy that error? :)

**Comment by ronf** [comment]
You should be able to use “pip install bcrypt" to install the “bcrypt” package (version 3.1.3 or later) to get what you need here. See https://asyncssh.readthedocs.io/en/latest/#optional-extras for more information on optional packages you can install.

You'll then need to provide the passphrase you used to create those keys in any calls you make to AsyncSSH which refers to them, at least in cases where you need access to a private key.

**Comment by AndKe** [comment]
thank you, please consider makeing the error message contain this information.  "pip install bcrypt" would be a nice clue :)

**Comment by ronf** [comment]
I'm hesitant to be that specific, as not everyone uses "pip" as their installation method. Some folks use packages from their Linux distribution, and others might use something like easy_install or even install packages manually using something like the setup.py script.

The error's mention of "KDF support" was because there were actually two different "bcrypt" packages on PyPI. Back then, "bcrypt" was the main package but it didn't support the KDF functionality needed by AsyncSSH, and so you had to install "py-bcrypt" instead for AsyncSSH to work. However, this was fixed in the bcrypt 3.0.0 release, eliminating the need to use the other version. As of 3.x, the main bcrypt package now works fine for AsyncSSH's needs, and AsyncSSH's optional dependencies were updated to reflect that.
